Library/Homebrew/cask/cmd.rb

```diff
@@ -54,13 +54,23 @@ class Cmd
     }.freeze
 
     DEPRECATED_COMMANDS = {
```

Comment:
Mind sticking a comment with `odeprecated` in it above here somewhere? That'll remind me to update these to `odisabled` for the minor release after next.

Reply:
There is a call to `odeprecated` somehwere below where they are passed.